src/banana/benchmark/external/apfel_utils.py

```diff
@@ -127,97 +134,102 @@ def load_apfel(theory, observables, pdf="ToyLH"):
     # apfel.SetGridParameters(3, 50, 3, 8e-1)
 
     # set APFEL grid to ours
-    if platform.node() in ["FHe19b", "topolinia-arch"]:
-        apfel.SetNumberOfGrids(1)
-        # create a 'double *' using swig wrapper
-        yad_xgrid = observables["interpolation_xgrid"]
-        xgrid = apfel.new_doubles(len(yad_xgrid))
+    apfel.SetNumberOfGrids(1)
+    # create a 'double *' using swig wrapper
+    yad_xgrid = observables["interpolation_xgrid"]
+    xgrid = apfel.new_doubles(len(yad_xgrid))
 
-        # fill the xgrid with
-        for j, x in enumerate(yad_xgrid):
-            apfel.doubles_setitem(xgrid, j, x)
+    # fill the xgrid with
+    for j, x in enumerate(yad_xgrid):
+        apfel.doubles_setitem(xgrid, j, x)
 
-        yad_deg = observables["interpolation_polynomial_degree"]
-        # 1 = gridnumber
-        apfel.SetExternalGrid(1, len(yad_xgrid) - 1, yad_deg, xgrid)
+    yad_deg = observables["interpolation_polynomial_degree"]
+    # 1 = gridnumber
+    apfel.SetExternalGrid(1, len(yad_xgrid) - 1, yad_deg, xgrid)
 
-    # set DIS params
+    # set pdf
     apfel.SetPDFSet(pdf)
-    apfel.SetProcessDIS(observables.get("prDIS", "EM"))
-    apfel.SetPropagatorCorrection(observables.get("PropagatorCorrection", 0))
-    apfel.SetPolarizationDIS(observables.get("PolarizationDIS", 0))
-    apfel.SetProjectileDIS(observables.get("ProjectileDIS", "electron"))
-    # set Target
 
-    # apfel initialization for DIS
-    apfel.InitializeAPFEL_DIS()
+    if prDIS:
+        # set Target
+        apfel.SetProcessDIS(observables.get("prDIS", "EM"))
+        apfel.SetPropagatorCorrection(observables.get("PropagatorCorrection", 0))
+        apfel.SetPolarizationDIS(observables.get("PolarizationDIS", 0))
+        apfel.SetProjectileDIS(observables.get("ProjectileDIS", "electron"))
+        # apfel initialization for DIS
+        apfel.InitializeAPFEL_DIS()
+    else:
+        # init evolution
+        apfel.InitializeAPFEL()
```

Comment:
keep the front-end specific stuff out of banana - simply call the rest in ekomark for example and add the ``apfel.InitializeAPFEL()`` there